src/components/exchange/SwapInfo.js

```diff
@@ -19,9 +17,15 @@ const Container = styled(Centered)`
   width: 100%;
 `;
 
-const SwapInfo = ({ asset, amount }) => {
+const SwapInfoCoinIcon = styled(CoinIcon).attrs({
+  size: 20,
+})`
+  flex: 0;
```

Comment:
why this?

Reply:
good question

Reply:
thanks for pointing this out! the `flex: 0` was undesired and was indeed breaking the layout!